VERSION

```diff
@@ -1,4 +1,6 @@
-v9.5.1
+v9.5.2
+
+v9.5.2 Bug fix in baggage options for JS client, cleanup tracing.go a bit
```

Comment:
I dont like that our SEMVER format restricts us to one line descriptions of change, but this is pretty vague.

Reply:
It doesn't, the release notes end up being the entirety of this file https://github.com/Clever/wag/releases

Reply:
👍  Updated with a slightly more verbose message.

Reply:
> It doesn't, the release notes end up being the entirety of this file https://github.com/Clever/wag/releases

Ah maybe I am thinking of ark